Package.swift

```diff
@@ -125,7 +130,10 @@ let package = Package(
 
         .target(
             name: "WorkflowReactiveSwift",
-            dependencies: ["ReactiveSwift", "Workflow"],
+            dependencies: [
+                "Workflow",
+                .product(name: "ReactiveSwift", package: "ReactiveSwift"),
```

Comment:
standardized some of these so now all the 'package-level' dependencies are just strings, and the external ones are `.product()`